main.tf

```diff
@@ -1152,7 +1152,7 @@ module "aws_load_balancer_controller" {
   # namespace creation is false here as kube-system already exists by default
   create_namespace = try(var.aws_load_balancer_controller.create_namespace, false)
   chart            = try(var.aws_load_balancer_controller.chart, "aws-load-balancer-controller")
-  chart_version    = try(var.aws_load_balancer_controller.chart_version, "1.4.8")
+  chart_version    = try(var.aws_load_balancer_controller.chart_version, "1.5.5")
```

Comment:
This is a breaking change. 
https://github.com/aws-ia/terraform-aws-eks-blueprints-addons/pull/190#issuecomment-1600629402

Reply:
what is breaking about this change?

Reply:
Chart 1.4.8 points to app version v2.4.7
Chart 1.5.0+ points to app version v2.5.0+
Problem is https://github.com/kubernetes-sigs/aws-load-balancer-controller/releases/tag/v2.5.0 has breaking changes. 
Also mentioned in the comment linked above pointing to my PR that I closed because of this.

Reply:
We don't consider third level dependencies in the semver of this module. Semver for this module is concerned only with the exposed interface of the module - users should not rely on the default chart versions of this module but instead pin to the version of their choosing. We will keep moving the module forward and applying updates as needed (such as this PR to update versions)